github-dark.css

```diff
@@ -1674,6 +1674,15 @@
     background: #222 !important;
     color: #777 !important;
   }
+  /*Create private repo Lock color*/
+  .page-new-repo .octicon-lock {
```

Comment:
Can you move this selector to the end of https://github.com/StylishThemes/GitHub-Dark/blob/6c61d52cc041c5b4693164754c6ab6ac716d3ff2/github-dark.css#L3449